src/api/app/models/binary_release.rb

```diff
@@ -57,7 +57,7 @@ def self.update_binary_releases_via_json(repository, json, time = Time.now)
           entry = existing.first
           if entry.binary_disturl                   == binary['disturl'] &&
              entry.binary_supportstatus             == binary['supportstatus'] &&
-             entry.binary_buildtime.to_datetime.utc == ::Time.at(binary['buildtime'].to_i).to_datetime.utc
+             entry.binary_buildtime.to_datetime.utc == ::Time.at(binary['buildtime'].to_i).try(:to_datetime).try(:utc)
```

Comment:
how can `at(binary['buildtime'].to_i)` return `nil`? Why is `::Time` written with `::`? Is this `Time` class the [default Ruby one](https://ruby-doc.org/core-2.5.0/Time.html#method-c-at)? :confused:

Reply:
I think I know it! :tada: What can be `nil` is in the other part of the `==`. `entry.binary_buildtime` can be `nil`, `Time.at(binary['buildtime'].to_i)` can never be `nil` :nerd_face: So this is not solving the bug :rofl: 

You could do for example:

```ruby
entry.binary_buildtime && entry.binary_buildtime.to_datetime.utc == Time.at(binary['buildtime'].to_i).to_datetime.utc

Reply:
Oh, right :flushed: